src/main/java/org/mifos/connector/mpesa/utility/MpesaUtils.java

```diff
@@ -61,25 +122,25 @@ public static String maskString(String strText) {
         int start = 0;
         int end = strText.length() - 4;
 
-        if (start > end){
+        if (start > end) {
             return "***";
         }
 
         int maskLength = end - start;
 
-        if(maskLength == 0)
+        if (maskLength == 0)
             return strText;
 
         StringBuilder sbMaskString = new StringBuilder(maskLength);
 
-        for(int i = 0; i < maskLength; i++){
+        for (int i = 0; i < maskLength; i++) {
             sbMaskString.append(maskChar);
         }
 
         return strText.substring(0, start) + sbMaskString + strText.substring(start + maskLength);
     }
 
-    public static void main(String[] args)  {
+    public static void main(String[] args) {
```

Comment:
Why do we need a main method here?  Donot sneak in main methods

Reply:
I have not added that. It was there previously.

Reply:
Lets remove that